src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Release History
 
 * Support Local context in acr task run
 
+**ARM**
+
+* Update azure-mgmt-resource package to use 6.0.0
```

Comment:
Please also specify the api version you used for Resource Type `MGMT_RESOURCE_POLICY`.